wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -5,7 +5,10 @@
 <html lang="{{ LANGUAGE_CODE }}" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}">
     <head>
         <meta charset="utf-8" />
-        <title>{% block branding_title %}Wagtail{% endblock %} - {% block titletag %}{% endblock %}</title>
+        <title>
+            {% block titletag %}{% endblock %} -
+            {% block branding_title %}Wagtail{% endblock %}
```

Comment:
I would recommend just keeping this in one line - it reads a bit better in the DOM output.

Reply:
Good call! I will change it to just one line.